foreign_cc/private/framework/platform.bzl

```diff
@@ -1,8 +1,40 @@
 """A helper module containing tools for detecting platform information"""
 
+SUPPORTED_CPU = [
+    "aarch64",
+    "s390x",
+    "x86_64",
+]
+
+SUPPORTED_OS = [
```

Comment:
Same comment as above but for the os values from platforms!

Reply:
yeah, see comment there, basically.